### PR TITLE
Allow admins to access bucket policies

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
This will help debug S3 access issues by confirming that the policy is formatted as expected.

### CODE REVIEW

File: s3-synapse-sync-bucket.j2 [hunk 1]
```diff
@@ -160,6 +160,7 @@ Resources:
             Action:
               - "s3:ListBucket*"
               - "s3:GetBucketLocation"
+              - "s3:GetBucketPolicy"
```
Comment (zaro0508) [inline s3-synapse-sync-bucket.j2:163]: seems fine, just curious why Admin access would just allow `s3:*` actions?

**Review by zaro0508 — APPROVED** [review]

File: s3-synapse-sync-bucket.j2 (hunk 1 above)
Comment (clarisse-lau) [reply s3-synapse-sync-bucket.j2:163]: sorry, I can't really recall... aside from maybe it didn't seem necessary at the time. 
Changing to `s3:*` for bucket-level access seems good to me

**Comment by BrunoGrandePhD** [comment]
@zaro0508: I broadened the admin permissions to `s3:*` so we don't need to add more permissions later. If you approve this, then feel free to merge and create a tag called `v1.6.1`.

**Comment by zaro0508** [comment]
@BrunoGrandePhD i have already approved.  You and @clarisse-lau are already maintainers of this repo so you have the power to do whatever you need.

**Pull request merged**